library/tests/src/com/google/maps/android/data/kml/KmlPolygonTest.java

```diff
@@ -5,6 +5,7 @@
 import junit.framework.TestCase;
 
 import java.util.ArrayList;
+import java.util.List;
 
 public class KmlPolygonTest extends TestCase {
```

Comment:
They fix a compilation issue I have on my side (possibly my java compiler is more up to date than yours?). Will file them as a separate pull request.